python/feldera/enums.py

```diff
@@ -36,155 +36,126 @@ class BuildMode(Enum):
 
 class PipelineStatus(Enum):
     """
-    Represents the state that this pipeline is currently in.
-
-    .. code-block:: text
-
-        Shutdown     ◄────┐
-        │         │
-        /deploy   │       │
-        │   ⌛ShuttingDown
-        ▼         ▲
-        ⌛Provisioning    │
-        │         │
-        Provisioned        │
-        ▼         │/shutdown
-        ⌛Initializing     │
-        │        │
-        ┌────────┴─────────┴─┐
-        │        ▼           │
-        │      Paused        │
-        │      │    ▲        │
-        │/start│    │/pause  │
-        │      ▼    │        │
-        │     Running        │
-        └──────────┬─────────┘
-                   │
-                   ▼
-                Failed
-    """
-
-    NOT_FOUND = 1
+     Represents the state that this pipeline is currently in.
+
+     .. code-block:: text
+
+                  Stopped ◄─────────── Stopping ◄───── All states can transition
+                     │                    ▲            to Stopping by either:
+    /start or /pause │                    │            (1) user calling /stop, or;
+                     ▼                    │            (2) pipeline encountering a fatal
+              ⌛Provisioning          Suspending            resource or runtime error,
+                     │                    ▲                having the system call /stop
+                     ▼                    │ /suspend       effectively
```

Comment:
State machine should be brought in-sync with the latest one which uses `/stop?force=false` instead of `/suspend`

Reply:
@abhizer